components/Explorer/Explorer.jsx

```diff
@@ -5,12 +5,9 @@ require('brace/theme/monokai');
 import fetch from 'isomorphic-fetch';
 import { HOST_URL } from '../../yasp.config';
 import RaisedButton from 'material-ui/RaisedButton';
+import TextField from 'material-ui/TextField';
```

Comment:
if those were unused then they would have caused a linting error so I would have removed them. Or are they in use in master at the moment?